roofit/xroofit/inc/RooFit/xRooFit/xRooBrowser.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "xRooNode.h"
```

Comment:
```suggestion
class xRooNode;
```
Can we turn this into a forward declaration, and then move all the function definitions of `xRooBrowser` that use the `xRooNode` to the source file?

That also means that the destructor has to be defined in the source file, even though it will be empty (because of the `std::shared_ptr<xRooNode>` data members).